### ISSUE TITLE
Request to add 1 important anti-fingerprint method.

### ISSUE THREAD

**Issue by Rikkit888** [kkapsner/CanvasBlocker#498]
Hey, this addon is really great.
Nearly there is no need of any other fingerprint addon, except for Javascript font fingerprint.
Against font fingerprint i use this really great addon:
https://addons.mozilla.org/de/firefox/addon/font-fingerprint-defender/
Sadly it has not much options and has only 1 mode to run.
It generates for each website a new font fingerprint.
Is it possible to implement this font anti-fingerprint method to CanvasBlocker and make it a persistent fingerprint, not changing all the time?

**Comment by kkapsner** [comment]
Font fingerprinting was already discussed in #200 (and #470) and there you can also find my opinion on font fingerprint defender: https://github.com/kkapsner/CanvasBlocker/issues/200#issuecomment-577720154

TLDR: it will not be implemented because I do not see a good way to do it.
